calour/database.py

```diff
@@ -363,8 +371,9 @@ def get_feature_terms(self, features, exp=None):
 
         Returns
         -------
-        feature_terms : dict of list of str/int
-            key is the feature, list contains all terms associated with the feature
+        feature_terms : dict of {feature(str): dict term_scores {term(str): score(float)}}
```

Comment:
this is hard to understand

Reply:
it's duplicated in the second PR (export_html) - fixed it there... sorry